packages/automation/index.js

```diff
@@ -456,5 +458,16 @@ textLine.each(function (d) {
     console.log('Modified files:', changedFiles.join(', '));
     fs.copyFileSync(pathFormHtml, destFormHtml);
   });
-
+  fs.readFile(pathMapHtml,'utf8',  function (err, html) {
```

Comment:
From this line script reads `publicgoods-website/map/index.html`. Then it splits and saves parts of html into multiple files. These files are used in `publicgoods-scripts/packages/map/pages/_document.js` to create a template page with wordpress styles.